examples/README.md

```diff
@@ -1 +1,146 @@
+
+## Set Up 
+
+### Docker
+
+Make sure Docker Desktop is running.
+
+### AWS Access Keys
+First make sure that your AWS access keys are configured in your root directory. To see if your credentials are setup run the following command:
+
+`cat .aws/credentials`
+
+If they are not configured, please visit [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) for instructions on how to set them up.
+
+### Grpc Installation
+
+In your root directory, run the following commands. These commands will take a while to install all the necessary components.
+
+`brew install PHP`
+
+`sudo pecl install grpc`
+
+If you are having issues please visit one of these resources:
+- https://github.com/grpc/grpc/tree/master/src/php
+- https://programmersought.com/article/22193951689/
+- https://grpc.io/docs/languages/php/quickstart/
+- https://cjyabraham.github.io/docs/quickstart/php/
+
+### Composer
+
+To check if you have composer installed, run the following command in your root directory:
+
+`composer -V`
+
+If the above does not work, please visit [here](https://getcomposer.org/download/) to install Composer. There are two methods, programmatically or manual file download. Please choose whichever way you prefer and then move the composer.phar file to your `$PATH` with the following command:
+
+`mv composer.phar /usr/local/bin/composer`
+
+### Update Repository Packages and Dependencies
+
+In the php contrib repository, run the following:
+
+`make install`
+
+Then run to make sure all dependencies and packages are up to date:
+
+`make update`
+
+Then run the following command to make sure the local composer is updated:
+
+`composer update`
+
+To make sure everything is working properly, run the following command to run all tests:
+
+`make install && make update && make style && make test && make phan && make psalm && make phpstan`
+
+At this point all necessary items have been installed in your system and you are ready to begin running the application.
+
+## Running the Applications
+
+### Run Collector
+
+Open a new terminal window and navigate into the aws-otel-collector folder. 
+
+Run the following command. Make sure to replace `YOUR_ACCESS_KEY_HERE` and `YOUR_SECRET_ACCESS_KEY_HERE` with your own specific keys.
+```
+docker run --rm -p 4317:4317 -p 55681:55681 -p 8889:8888 \
+   -e AWS_REGION=us-west-2 \
+   -e AWS_ACCESS_KEY_ID=YOUR_ACCESS_KEY_HERE \
+   -e AWS_SECRET_ACCESS_KEY=YOUR_SECRET_ACCESS_KEY_HERE \
+   -v "${PWD}/examples/docker/config-test.yaml":/otel-local-config.yaml \
+   --name awscollector public.ecr.aws/aws-observability/aws-otel-collector:latest \
+   --config otel-local-config.yaml
+```
+In another terminal window, navigate to the opentelemetry-php-contrib folder.
+
+Run the following command for Sample App 1:
+
+`php examples/aws/SampleApp1/SampleApp1.php`
+
+The output for this app should look similar to the following:
```

Comment:
Should you mention that the aws-sdk-call is not implemented yet? Not necessary but might help ease some confusion